lzy/worker/docker/Worker.Base.Dockerfile

```diff
@@ -4,7 +4,7 @@ ENV DEBIAN_FRONTEND noninteractive
 
 ### deps
 RUN apt-get -y update && \
-    apt-get -y install fuse lsof procps curl bash tar wget && \
+    apt-get -y install fuse lsof procps curl bash tar wget locales && \
```

Comment:
Why wouldn't you bring this shared code to a new base image?

Reply:
Our 3 current base images has 2 different FROM images. Our 2 base images with common FROM cuda image has not so much common code. I think it's not necessary.